libraries/ESP8266WiFi/examples/NTPClient/NTPClient.ino

```diff
@@ -21,8 +21,13 @@
 #include <ESP8266WiFi.h>
 #include <WiFiUdp.h>
 
-char ssid[] = "*************";  //  your network SSID (name)
-char pass[] = "********";       // your network password
+#ifndef STASSID
+#define STASSID "your-ssid"
+#define PSK  "your-password"
+#endif
+
+char ssid[] = STASSID; // your network SSID (name)
```

Comment:
prior examples were ssid*, but here it0s ssid[]. Same for pass right below.

Reply:
Except from WiFiTelnetToSerial and axTLS examples which have bigger changes,
I only added `APSSID`/`APPSK` defines to all the others without much further updates.
There is no general template on how they use ssid and password.

Reply:
Yes, we're bad there since dozens of people have submitted these things.  `const` is appropriate here, too.

Don't get me started on `DEBUG`-type macros, they're a complete mess that makes this look like nothing!